sites/partners/src/listings/PaperListingForm/sections/BuildingDetails.tsx

```diff
@@ -66,6 +66,14 @@ const neighborhoodOptions: SelectOption[] = [
   { value: "West End area", label: "West End area" },
 ]
 
+const regionOptions: SelectOption[] = [
+  { value: "", label: "" },
+  ...Object.values(Region).map((item) => ({
+    label: item,
+    value: item,
+  })),
+]
+
```

Comment:
There's at least one other place we transform an object into options like in ApplicationTypes and we have an `arrayToFormOptions` function in helpers. Could you please create an `objectToFormOptions` function along the same lines as arrayToFormOptions.